sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -169,8 +169,8 @@ internal object KlaviyoApiClient : ApiClient {
                     } catch (exception: JSONException) {
                         wasMutated = true
                         Registry.log.wtf("Invalid request JSON $uuid", exception)
+                        Registry.log.info(json)
```

Comment:
Just double checking here...you changed the order of operations but this seems consistent with the block above so I think it's fine.